components/brave_rewards/resources/ui/storage.ts

```diff
@@ -53,7 +53,8 @@ export const defaultState: Rewards.State = {
     adsPerHour: 0,
     adsUIEnabled: false,
     adsNotificationsReceived: 0,
-    adsEstimatedEarnings: 0
+    adsEstimatedEarnings: 0,
+    adsIsSupported: true
```

Comment:
I think we should start with default false as ads are only supported in few countries

Reply:
done